doc/HOWTO-KAFKA.md

```diff
@@ -4,6 +4,8 @@ In the "daemon mode", Sonar stays memory-resident and pushes data to a network s
 sinks is a Kafka broker.  See HOWTO-DAEMON.md for general information about this mode and the
 options available for configuring the Kafka producer in Sonar.
 
+See `../tests/kafka/` for examples of config files for various node and master types.
```

Comment:
This can be moved to the first patch because the ini files are being moved there.